src/components/form/index.tsx

```diff
@@ -0,0 +1,148 @@
+import PopUp from '#components/pop-up'
+import styled from '@emotion/styled'
+import React from 'react'
+
+function Feedback() {
+  const [isDisplayed, setDisplay] = React.useState(false)
+
+  function handleSubmit(e: React.FormEvent<HTMLFormElement>): void {
+    e.preventDefault()
+    const {
+      name,
+      raidName,
+      rate,
+      description,
+    } = e.target as typeof e.target & {
+      name: { value: string }
+      raidName: { value: string }
+      rate: { value: string }
+      description: { value: string }
+    }
+    console.log({
+      name: name.value,
+      raidName: raidName.value,
+      rate: rate.value,
+      description: description.value,
+    })
+  }
+
+  return (
+    <>
```

Comment:
Prefer Wrapping with containers, even a `div` is fine. Makes working with flex and layouting much easier

Reply:
hmmm I agree but here i used a fragment because i didnt want any css bugs/overlapping

Reply:
fragments cause css bugs from my experience, can you elaborate how they reduce them?

Reply:
hmm that's something i never faced. but i can say the same for divs thu
Fragments cant be styles because they are hidden for the browsers.

Reply:
The problem is the following
```jsx
<div>
  <Thing />
  <AnotherThing />
  <ThingTheThird />
<div>
```
let's say I want to order these elements in a row and not a column.
```jsx
<div style={{display: 'flex' }}>
  <Thing />
  <AnotherThing />
  <ThingTheThird />
<div>
```
now instead of having 3 elements in a row, as I would if they all were wrapped in a `div`, now I have N elements in a row, because the objects are effected by the display:flex of the ancestor. This makes styling a cascading nightmare!

Reply:
It's not an issue, let's not prematurely solve problems we don't have.

Reply:
Currently it's a none-issue, but one day it'll be tech-debt. But whatever you guys prefer